tensorflow_serving/sources/storage_path/file_system_storage_path_source_test.cc

```diff
@@ -171,6 +171,40 @@ TEST(FileSystemStoragePathSourceTest, MultipleVersions) {
                    .PollFileSystemAndInvokeCallback());
 }
 
+TEST(FileSystemStoragePathSourceTest, MultipleVersionsAtTheSameTime) {
+  const string base_path = io::JoinPath(testing::TmpDir(), "MultipleVersionsAtTheSameTime");
+  TF_ASSERT_OK(Env::Default()->CreateDir(base_path));
+  TF_ASSERT_OK(Env::Default()->CreateDir(
+      io::JoinPath(base_path, "non_numerical_child")));
+  TF_ASSERT_OK(Env::Default()->CreateDir(io::JoinPath(base_path, "42")));
+  TF_ASSERT_OK(Env::Default()->CreateDir(io::JoinPath(base_path, "17")));
+
+  string all_version_policy = std::to_string(FileSystemStoragePathSourceConfig::ALL_VERSIONS);
```

Comment:
I don't think you need to do this. You can simply in-line ALL_VERSIONS int he proto text, as in:

servables {
  version_policy: ALL_VERSIONS
  ...